usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -436,36 +438,21 @@ def construct_es_response_for_prime_awards(self, response) -> dict:
             ],
         }
 
-    def append_recipient_hash_level(self, result) -> dict:
```

Comment:
What happened to this code to make it go away? What replaced it to satisfy its function?

Reply:
This was replaced with `get_recipient_hash_with_level`. Instead of running an individual DB query per each Award record we get back from Elasticsearch it is now grabbing the Recipient Hash and the first level from the result as we step through the results.